v22.2/install-cockroachdb-linux.md

```diff
@@ -49,17 +42,17 @@ Use one of the options below to install CockroachDB.
           </div>
           <div class="highlight"><pre><code class="language-shell" data-lang="shell"><span class="nb">mkdir</span> <span class="nt">-p</span> /usr/local/lib/cockroach</code></pre></div>
         </li>
-        <li>Copy the library files to the directory:</p>
+        <li>Copy the library files to the directory. In the following commands, replace `{ARCHITECTURE}` with `linux-amd64` for Intel, or with `linux-arm64` for ARM.</p>
```

Comment:
```suggestion
         <li>Copy the library files to the directory. In the following commands, replace `{ARCHITECTURE}` with `linux-amd64` for Intel, or with `linux-3.7.10-gnu-aarch64` for ARM.</p>
```

For example, the file name for the alpha.4 archive is `cockroach-v22.2.0-alpha.4.linux-3.7.10-gnu-aarch64.tgz`.